test/onnx/test_op_consistancy.py

```diff
@@ -0,0 +1,460 @@
+# Owner(s): ["module: onnx"]
+
+"""Test consistency between torch.onnx exported operators and torch operators.
```

Comment:
This file has a slight misspelling in its name, it would be "test_op_consistency.py"